packages/web-twig/DEPRECATIONS-v3.md

```diff
@@ -4,7 +4,166 @@ Following deprecations will be removed in version 3 of the _spirit-web-twig_ pac
 
 > Please follow these steps to safely upgrade your design system to Spirit Design System v3 components.
 
-## Add the `danger` icon to your project's assets
+[What are deprecations?][readme-deprecations]
+
+## Deprecations
+
+### Required `id` prop for form components
+
+The `id` prop will be required for all form components.
+
+Related components:
+
+- `Checkbox`
+- `HelperText`
+- `Radio`
+- `Select`
+- `ValidationText`
+
+#### Migration Guide
+
+Add `id` prop to the form components.
+
+### Dropdown `id` prop
+
+The `id` prop will be mandatory for the `Dropdown` component.
+
+#### Migration Guide
+
+Add `id` prop to the `Dropdown` component.
+
+### Tooltip and Dropdown Placements
+
+The `Tooltip` and `Dropdown` components no longer support non-flows relative placements.
+
+#### Migration Guide
+
+Instead of using `top-left` or `left-bottom` and etc, use `top-start` or `left-end` and so on.
+See [Placement dictionary][dictionary-placement] for more details.
+
+- `<Tooltip placement="top-left" … />` → `<Tooltip placement="top-start" … />`
+- `<Tooltip placement="top-right" … />` → `<Tooltip placement="top-end" … />`
+- `<Tooltip placement="right-top" … />` → `<Tooltip placement="right-start" … />`
+- `<Tooltip placement="right-bottom" … />` → `<Tooltip placement="right-end" … />`
+- `<Tooltip placement="bottom-left" … />` → `<Tooltip placement="bottom-start" … />`
+- `<Tooltip placement="bottom-right" … />` → `<Tooltip placement="bottom-end" … />`
+- `<Tooltip placement="left-top" … />` → `<Tooltip placement="left-start" … />`
+- `<Tooltip placement="left-bottom" … />` → `<Tooltip placement="left-end" … />`
+- `<Dropdown placement="top-left" … />` → `<Dropdown placement="top-start" … />`
+- `<Dropdown placement="top-right" … />` → `<Dropdown placement="top-end" … />`
+- `<Dropdown placement="right-top" … />` → `<Dropdown placement="right-start" … />`
+- `<Dropdown placement="right-bottom" … />` → `<Dropdown placement="right-end" … />`
+- `<Dropdown placement="bottom-left" … />` → `<Dropdown placement="bottom-start" … />`
+- `<Dropdown placement="bottom-right" … />` → `<Dropdown placement="bottom-end" … />`
+- `<Dropdown placement="left-top" … />` → `<Dropdown placement="left-start" … />`
+- `<Dropdown placement="left-bottom" … />` → `<Dropdown placement="left-end" … />`
+
+### Tooltip composition
```

Comment:
Either Title Case or Sentence case should be used for headings. A good tip for our MD linter.

Reply:
I think that `Title Case` will be required: https://github.com/lmc-eu/code-quality-tools/blob/main/.textlintrc.js#L4